viewer/src/main/webapp/WEB-INF/jsp/app.jsp

```diff
@@ -99,8 +99,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                 </c:otherwise>
             </c:choose>
         </c:if>
-        <script type="text/javascript" src="${contextPath}/viewer-html/common/overrides.js"></script>
-        <script type="text/javascript" src="${contextPath}/resources/js/FileSaver.js"></script>
+                  
+        <c:if test="${actionBean.viewerType == 'ol'}">
+               <link href="${contextPath}/viewer-html/common/ol/ol.css" rel="stylesheet">  
+               <link href="${contextPath}/viewer-html/common/resources/css/openlayers.css" rel="stylesheet">
+               <script src="http://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.6/proj4.js" type="text/javascript"></script>
```

Comment:
this should be using https otherwise loading will fail on https sites (using https will not fail on http sites). maybe better to put this file in the repo so it is available locally